studio/components/ui/PageTelemetry.tsx

```
@@ -54,6 +54,13 @@ const PageTelemetry: FC = ({ children }) => {
           language: telemetryProps?.language,
         },
       })
+
+      post(`${API_URL}/telemetry/pageview`, {
```

Assuming `post` is async and we're not waiting to this fails silently?